cmd/vela-hugo/hugo.go

```diff
@@ -118,7 +120,7 @@ func install(extendedBinary bool, customVer, defaultVer string) error {
 	}
 
 	// create the download URL to install hugo - https://github.com/gohugoio/hugo/releases
-	url := fmt.Sprintf(_download, customVer, binary, customVer, osName, archType)
+	url := fmt.Sprintf(_download, ver.String(), binary, ver.String(), osName, archType)
```

Comment:
Should we be re-using the `verWithoutV` variable here instead of calling `ver.String()` again?

Reply:
100% forgot to update - thanks

Reply:
update pushed